src/app/(payload)/api/seat-holding/seat/utils/index.ts

```
@@ -0,0 +1,47 @@
+      limit: 1000,
+    })
+    .then((res) => res.docs)
+    .catch((error) => {
+      console.error('Error while existingSeats ', error)
+
+      return []
+    })
+
+  return existingSeats
+    .map((ext) => (ext.seatName || '').split(','))
+    .reduce((arr, arrItem) => [...arrItem, ...arr], [])
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)